<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Screenshot-tray Prototype</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="screenshotBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>900</width>
      <height>600</height>
     </rect>
    </property>
    <property name="title">
     <string>Screenshot</string>
    </property>
    <widget class="QGraphicsView" name="screenshotView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>881</width>
       <height>561</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>620</y>
      <width>501</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="shootButton">
       <property name="text">
        <string>&amp;Shoot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>S&amp;ave</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="hideButton">
       <property name="text">
        <string>&amp;Hide</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="text">
        <string>&amp;Quit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
